tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -771,6 +770,44 @@ void RunCppCheck(string projectName, MSBuildTargetPlatform platform)
         });
 
 
+    Target BuildProfilerTsanTest => _ => _
```

Comment:
So this rebuilds everything e.g. native wrapper, native loder, just wondering if that's necessary? 🤔 May well not be worth _not_ doing this, just wondering?